include/miniocpp/config.h

```diff
@@ -21,8 +21,8 @@
 #define MINIO_CPP_STRINGIFY(x) #x
 
 #define MINIO_CPP_MAJOR_VERSION 0
-#define MINIO_CPP_MINOR_VERSION 1
-#define MINIO_CPP_PATCH_VERSION 1
+#define MINIO_CPP_MINOR_VERSION 2
+#define MINIO_CPP_PATCH_VERSION 0
```

Comment:
Is it possible to generate this file at the time of build?

Reply:
It would be possible to generate this file, but I would rather keep it as is as now we don't have generated headers, which should be the preferred case.